client/src/util.c

```diff
@@ -207,3 +208,22 @@ void util_rotatePoint(struct DoublePoint *point)
       break;
   }
 }
+
+bool util_hasGLExt(const char * exts, const char * ext)
```

Comment:
This is simply a word-based text search, perhaps `util_containsWord` or a similar non-GL specific name?

Reply:
This function is not really a general word search function because it only works on space separated lists. If it was called `util_containsWord`, I would expect it to handle punctuation, which is not the case here.

It does not make sense to generalize this function because OpenGL extension lists are separated only by spaces, so I think this function should stay as is.

Reply:
true, however, I am not keen on adding GL-specific methods (even if just in name) to the `common` project, it's intended to be generic utility functions and code that is not specific to any particular technology. IMO it would be better to name this `util_containsValue` with the delimiter being an argument making this a generic string method, ie:

`bool util_containsValue(const char * list, const char delimiter, const char * value)`

Reply:
This is in the client project, not the common project.

Reply:
Ah, you're correct, in which case IMO it should be in `common/src/stringutils.c` and then have the GL specific version in utils.c which calls the generic version in stringutils.

Reply:
btw, the reason being that the OBS client or even the host may at some point find use in this method, best to avoid code duplication.